### PR TITLE
Add support for Django 2.x drop support for Django 1.x

### LINKED ISSUE #21: Test against Django 2.0

### PR DESCRIPTION
**Pull request by chrisseto**
Closes https://github.com/chrisseto/django-include/issues/21

cc @Johnetordoff

Tests pass locally, tox seems to be having some troubles.

### CODE REVIEW

**Review by sloria — APPROVED** [review]
This is nice cleanup 👍

File: tox.ini [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
-    django19: Django>=1.9,<1.10
-    django110: Django>=1.10,<1.11
-    django111: Django>=1.11,<1.12
+    django2x: Django>=2.0,<3.0
+    django20: Django>=2.0,<2.1
+    django21: Django>=2.1,<3.0
 
-commands=inv test {posargs}
+commands=
+  faster: pip install -e .[faster]
```
Comment (sloria) [inline tox.ini:19]: This is fine, but I think the tox-y way to do is like

```
[testenv]
extras =
    faster: faster
```

File: tox.ini [hunk 2]
```diff
@@ -1,18 +1,20 @@
 [tox]
-minversion = 2.1
-envlist = py{27,35,36}-django{19,110,111}
+minversion = 3.1
+envlist = py{36,37,38}-django{2x,20,21}{,-faster}
```
Comment (sloria) [inline tox.ini:3]: Unimportant:

```suggestion
envlist = py{36,37,38}-django{2x,20,21}-{,faster}
```

Comment (chrisseto) [reply tox.ini:3]: Is that correct?
I would expect the outputs of  `django20{,-faster}` to be `django20` and `django20-faster`
and the outputs of `django20-{,faster}` to be `django20-` and `django20-faster`

I also haven't used tox in a while so I may be misunderstanding what the expectations are.

File: tox.ini (hunk 1 above)
Comment (chrisseto) [reply tox.ini:19]: TIL, thanks!

File: tox.ini (hunk 2 above)
Comment (sloria) [reply tox.ini:3]: You're correct

```
~/projects/forks/django-include django2
❯ tox -l
py36-django2x
py36-django2x-faster
py36-django20
py36-django20-faster
py36-django21
py36-django21-faster
py37-django2x
py37-django2x-faster
py37-django20
py37-django20-faster
py37-django21
py37-django21-faster
py38-django2x
py38-django2x-faster
py38-django20
py38-django20-faster
py38-django21
py38-django21-faster

~/projects/forks/django-include django2
❯ tox -l
py36-django2x-
py36-django2x-faster
py36-django20-
py36-django20-faster
py36-django21-
py36-django21-faster
py37-django2x-
py37-django2x-faster
py37-django20-
py37-django20-faster
py37-django21-
py37-django21-faster
py38-django2x-
py38-django2x-faster
py38-django20-
py38-django20-faster
py38-django21-
py38-django21-faster
```

but tox just ignores the trailing `-` as a noop.

That said, I think what you have here is better. It's weird seeing the trailing `-` in the `tox -l`. Disregard my suggestion

**Comment by Johnetordoff** [comment]
@chrisseto @sloria  We good?

**Comment by chrisseto** [comment]
FWIW, this also appears to work on Django 3.0

**Pull request merged**